theme/templates/accounts/profile.html

```diff
@@ -662,13 +662,13 @@ <h3 class="panel-title"><span
                                             <tr>
                                                 <td>
                                                     {% if name == "GoogleScholarID" %}
-                                                    <img class="identifier-icon ai ai-google-scholar-square"/>
+                                                    <img class="identifier-icon ai ai-google-scholar-square" src="{% static 'img/google-scholar.svg' %}"/>
```

Comment:
Why are these changes made? How did this work before you introduced source?

Reply:
solid question, it's not obvious!

Before, we had the images defined in our css, with hard links to `static/static`:
https://github.com/hydroshare/hydroshare/pull/5482/files#diff-94565d8a46949822da99c6938042a6414adf62d4d327dc337836b3d20f3f5727L69-L86

Getting an argument for the static url into css requires a whole bunch more in our pipeline (css bundler, sass, etc) so instead, I define the `src` for the image in the django template.